TCAT/Cells/LargeDetailTableViewCell.swift

```diff
@@ -136,61 +136,60 @@ class LargeDetailTableViewCell: UITableViewCell {
             label.text! += " "
             label.sizeToFit()
         }
-        
+
         // Format and place labels
-        
+
         let content = label.text! + direction.locationNameDescription
         let attributedString = bold(pattern: direction.name, in: content)
         label.attributedText = attributedString
         paragraphStyle.lineSpacing = 4
-        
+
         label.numberOfLines = 0
         label.sizeToFit()
         label.frame.size.width = (chevron.frame.minX - 12) - cellWidth
         label.frame.origin.y = edgeSpacing // - paragraphStyle.lineSpacing
-        
+
         attributedString.addAttribute(NSAttributedStringKey.paragraphStyle,
                                       value: paragraphStyle,
-                                      range: NSMakeRange(0, label.attributedText!.length))
+                                      range: NSRange(location: 0, length: label.attributedText!.length))
         label.attributedText = attributedString
-        
+
         return label
     }
-    
+
     /** Abstracted formatting of content for detailLabel. Needs titleLabel */
     func formatDetailLabel(_ label: UILabel, _ titleLabel: UILabel) -> UILabel {
-        
+
         label.text = "\(direction.stops.count + 1) stop\(direction.stops.count + 1 == 1 ? "" : "s")"
-        
+
         // Number of minutes for the bus direction
         var timeString = Time.timeString(from: direction.startTime, to: direction.endTime)
         if timeString == "0 min" {  timeString = "1 min" }
```

Comment:
better to check on an int rather than string in case we change the format of our strings. i think we should have a method in our `Time` class that does that but could be wrong